nemo-physical/src/table_reader.rs

```diff
@@ -4,7 +4,7 @@
 use crate::{builder_proxy::PhysicalBuilderProxyEnum, error::ReadingError};
 
 /// Resource that can be reference in source declarations in Nemo programs
-/// Resources are resolved using `nemo::io::input_manager::ResourceProviders`
+/// Resources are resolved using `nemo::io::resource_providers::ResourceProviders`
```

Comment:
I guess this should be a link (if it were, we would have gotten a warning already for the target not existing)